packages/@aws-cdk/aws-ecs/lib/ec2/ec2-service.ts

```diff
@@ -198,6 +199,48 @@ export class Ec2Service extends BaseService implements elb.ILoadBalancerTarget {
     });
   }
 
+  /**
+   * Enable CloudMap service discovery for the service
+   */
+  public enableServiceDiscovery(props: ServiceDiscoveryProps): cloudmap.Service {
```

Comment:
Does use of this method require that the Cluster has `addNamespace()` called?

Reply:
I am, please do.

Reply:
By the way, I think we're standardizing on `FooOptions` for arguments to a method (`FooProps` is for arguments to a constructor)